optuna/samplers/_tpe/sampler.py

```diff
@@ -893,3 +891,41 @@ def _calculate_weights_below_for_multi_objective(
     weights_below_all[feasible_mask] = weights_below
     weights_below_all = weights_below_all[~np.isnan(cvals)]
     return weights_below_all
+
+
+def _clamp_inf_and_calc_reference_point(lvals: np.ndarray) -> Tuple[np.ndarray, np.ndarray]:
```

Comment:
I'm not sure but "clip" instead of "clamp" is true?

Reply:
+1 with @HideakiImamura  since NumPy uses [`clip`](https://numpy.org/doc/stable/reference/generated/numpy.clip.html).